prow/githubeventserver/options.go

```diff
@@ -29,14 +29,14 @@ import (
 type Options struct {
 	// HmacTokenGenerator is a function that holds a hmac token that will be used
 	// in a github event server
-	HmacTokenGenerator func() []byte
+	hmacTokenGenerator func() []byte
```

Comment:
What do we gain from unexporting these and using setters?

Reply:
I don't think that those fields should be exposed. I would prefer to allow to user to set his values with a helper function instead. Also, those values will be added by the user, so technically they are already exposed to the user.

Reply:
Sorry I don't see any value in having a setter like this that only exists to write unconditionally to a private field. Having this setter doesn't make the field any less exposed, it just makes it less convenient to set.